en/docs/GettingStarted/quick-start-guide.md

```diff
@@ -1,123 +1,182 @@
+        will be using the service URL (`https://www.mocky.io/v2/5185415ba171ea3a00704eed`) which is shown 
+        in the mock service.
+
+     ![]({{base_path}}/assets/img/get_started/Mocky_io.png)
+         
+    * Optionally, to test this service, copy the service URL 
+        [https://www.mocky.io/v2/5185415ba171ea3a00704eed](https://www.mocky.io/v2/5185415ba171ea3a00704eed) on a new browser and navigate to it. You 
+        should see a JSON message saying 
+            
+            `{"hello": "world"}`. 
     
-    ![](../assets/img/get_started/design_new_rest_api.png)
+4. On the API Publisher Portal, click on the `Create New API` dropdown and choose the `Design a new REST API` option.
+   
+    ![]({{base_path}}/assets/img/get_started/design_new_rest_api.png)
 
 
-6.  Fill in the following details in the form that appears. Use the URL from mocky.io as the endpoint. Note to use the http protocol for now since using https for mocky.io requires importing its certificate into API Manager.
+5. Fill in the following details in the form that appears. Use the URL we obtained at step3 from mocky.io as the endpoint.
 
-    ![](../assets/img/get_started/api_create.png)  
+     
+    ![]({{base_path}}/assets/img/get_started/api_create.png)  
+         
+    !!! important
+        __Use the http protocol__ since using https for mocky.io requires importing its certificate  
+        into API Manager 
+        
+6. Click on the `Create & Publish` button once done. This will publish your first API on the developer portal as well as deploy it on the API gateway. You now have an OAuth2.0 secured REST API that is ready to be consumed.
 
-7.  Click on the `Create & Publish` button once done. This will deploy your first API on the developer portal and API gateway. You now have an OAuth2.0 secured REST API that is ready to be consumed.
+#### Subscribe to the API through the developer portal and generate keys
 
-### Consuming the API from the developer portal.
+1. Open the developer portal [https://localhost:9443/devportal](https://localhost:9443/devportal). The published `HelloWorld` API should be listed on the developer portal as shown below.
 
-1.  Open up a new browser terminal and navigate to `https://localhost:9443/devportal`. You should see the `HelloWorld` API appearing on the portal. Click on the `Sign-In` button on the top right corner to login to the developer portal and sign-in by entering the default **`admin/admin`** credentials.
+    ![]({{base_path}}/assets/img/get_started/dev_portal_landing_page.png)
 
-2. Once logged in click on the API thumbnail to view details of the API. You should see a page that gives you an overview of the API. Click on the button that says `Key Generation Wizard`. This will walk you through a few steps that registers an OAuth2.0 application which consumes this API.  
+2. Click the `Sign-In` button on the top right corner to Sign-In to the developer portal with    **`admin/admin`** credentials.
```

Comment:
This can be shortened to the following.

Click `Sign-In` and enter   **`admin/admin`** as your credentials to sign in.